ci/travis.rb

```diff
@@ -145,7 +145,7 @@ def run_bug_report_templates
 
 ENV['GEM'].split(',').each do |gem|
   [false, true].each do |isolated|
-    next if ENV['TRAVIS_PULL_REQUEST'] && ENV['TRAVIS_PULL_REQUEST'] != 'false' && isolated
```

Comment:
Haha 😄 

Why don't you send PR for this in Rails own repo? You created PR in your repo.